README.md

```diff
@@ -1,130 +1,43 @@
-
-1. Create a pull request with your code, targeting the master branch of the repository golang-bootcamp-2020.
-2. Fill this [form](https://forms.gle/ogQtHBk6DtZ5yKUM9) including the PR’s url
-3. Stay tune for feedback
-4. Do the changes according to your mentor's comments
-
-## Documentation
-
-### Must to learn
-
-- [Go Tour](https://tour.golang.org/welcome/1)
-- [Go basics](https://www.youtube.com/watch?v=C8LgvuEBraI)
-- [Git](https://www.youtube.com/watch?v=USjZcfj8yxE)
-- [Tool to practice Git online](https://learngitbranching.js.org/)
-- [Effective Go](https://golang.org/doc/effective_go.html)
-- [How to write code](https://golang.org/doc/code.html)
-- [Go by example](https://gobyexample.com/)
-- [Go cheatsheet](http://cht.sh/go/:learn)
-- [Any talk by Rob Pike](https://www.youtube.com/results?search_query=rob+pike)
-- [The Go Playground](https://play.golang.org/)
-
-### Self-Study Material
-
-- [Golang Docs](https://golang.org/doc/)
-- [Constants](https://www.youtube.com/watch?v=lHJ33KvdyN4)
-- [Variables](https://www.youtube.com/watch?v=sZoRSbokUE8)
-- [Types](https://www.youtube.com/watch?v=pM0-CMysa_M)
-- [Functions](https://www.youtube.com/watch?v=feU9DQNoKGE)
-- [Error Handling](https://www.youtube.com/watch?v=26ahsUf4sF8)
-- [Modules](https://www.youtube.com/watch?v=Z1VhG7cf83M)
-  - [Part 1 and 2](https://blog.golang.org/using-go-modules)
-- [Go tools](https://dominik.honnef.co/posts/2014/12/an_incomplete_list_of_go_tools/)
-- [More Go tools](https://dev.to/plutov/go-tools-are-awesome-bom)
-- [Clean Architecture](https://medium.com/@manakuro/clean-architecture-with-go-bce409427d31)
-- [For Loops](https://www.youtube.com/watch?v=0A5fReZUdRk)
-- [Arrays and Slices](https://www.youtube.com/watch?v=d_J9jeIUWmI)
-- [Conditional statements: If](https://www.youtube.com/watch?v=QgBYnz6I7p4)
-- [Multiple options conditional: Switch](https://www.youtube.com/watch?v=hx9iHend6jM)
-- [Maps](https://www.youtube.com/watch?v=p4LS3UdgJA4)
-- [Structures](https://www.youtube.com/watch?v=w7LzQyvriog)
-- [Structs and Functions](https://www.youtube.com/watch?v=RUQADmZdG74)
-- [Pointers](https://tour.golang.org/moretypes/1)
-- [Interfaces](https://tour.golang.org/methods/9)
-- [Interfaces](https://gobyexample.com/interfaces)
-- [Methods](https://www.youtube.com/watch?v=nYWa5ECYsTQ)
-- [Failed requests handling](http://www.metabates.com/2015/10/15/handling-http-request-errors-in-go/)
-- [Packages](https://www.youtube.com/watch?v=sf7f4QGkwfE)
-- [Unit testing](https://golang.org/pkg/testing/)
-- [Functions as values](https://tour.golang.org/moretypes/24)
+First Deliverable (due November 22th 23:59PM)
+Based on the self-study material and mentorship covered until this deliverable, I have perform the following:
+
+Select architecture
+Read a CSV
+Handle Errors for CSV  (not valid, missing)
+Use best practices
+Use lint, error check
+
+## Run
+Clone the repository and execute the following command on the root folder.
+``` bash
```

Comment:
By convention, every code block should be separated from the paragraph above by an empty line